terracumber_config/tf_files/SUSEManager-Head-NUE.tf

```diff
@@ -194,18 +194,18 @@ module "cucumber_testsuite" {
       install_salt_bundle = true
     }
     pxeboot-minion = {
-      image = "sles15sp3o"
+      image = "sles15sp4o"
```

Comment:
Maybe yes, maybe not. I don't know about support for SP4 terminals in Retail.

(it is reinstalled anyway, but for idempotency reasons, we prefere to have same version before and after the reinstallation)

Reply:
Vladimir clarified that SP4 should work in Retail.

Reply:
Didn`t I do that already in https://github.com/uyuni-project/uyuni/pull/5413?

Reply:
Is this still relevant @Bischoff?